cli/ballerina-cli/src/main/resources/create_cmd_templates/service/tests/service_test.bal

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import ballerina/io;
+import ballerina/http;
+import ballerina/test;
+
+http:Client testClient = check new ("http://localhost:9090/hello");
+
+# Before Suite Function
+
+@test:BeforeSuite
+function beforeSuiteFunc() {
+    io:println("I'm the before suite function!");
+}
+
+# Test function
+
+@test:Config {}
+function testServiceFunction() {
+    string|error response = testClient->get("/sayHello/?name=John");
+    test:assertEquals(response, "Hello, John");
+}
+
+# Negative test function
+
+@test:Config {}
+function testServiceFunctionNegative() returns error? {
```

Comment:
This is not a negative case. The service should return an error and the test case should be updated accordingly.